rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/execution/RdScheduledExecutionStats.java

```diff
@@ -0,0 +1,7 @@
+package org.rundeck.app.data.model.v1.execution;
+
+public interface RdScheduledExecutionStats {
+
```

Comment:
Add a field for the id reference to the job. Something like `Serializable getJobId()`